mage_ai/frontend/components/PipelineDetail/AddNewBlocks/index.tsx

```diff
@@ -7,21 +7,16 @@ import KeyboardShortcutButton from '@oracle/elements/Button/KeyboardShortcutButt
 import Spacing from '@oracle/elements/Spacing';
 import { Add } from '@oracle/icons';
 import { AxisEnum } from '@interfaces/ActionPayloadType';
-import { BlockRequestPayloadType, BlockTypeEnum } from '@interfaces/BlockType';
+import { BlockRequestPayloadType, BlockTypeEnum, BLOCK_TYPE_CONVERTIBLE } from '@interfaces/BlockType';
```

Comment:
@shrey-mage missed this in the other PR, but rename `BLOCK_TYPE_CONVERTIBLE` to `CONVERTIBLE_BLOCK_TYPES`.